bluebottle/tasks/templates/tasks/mails/task_status_closed.mail.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.mail.html" %}
+{% load i18n %}
+
+{% block content %}
+    {% blocktrans with receiver_name=to.short_name %}
+
+        Hello {{ receiver_name }}
+        TODO: Fix content
```

Comment:
Guess we just have to create a ticket for this. Make sure we add all the mails that need content.